docs/user-guide/api-mediation/configuration-at-tls.md

```diff
@@ -327,3 +308,294 @@ An error can occur if the [list of ciphers](#ciphers) does not match between the
 
 **Solution:**  
 Review the supported TLS versions and ciphers used in both the client and the server.
+
+## Full example of AT-TLS configuration
+
+Review a full working example of an AT-TLS configuration file on z/OS, specifically used for defining secure communication between different services in a mainframe environment. All port values are examples.
+The example is commented for convenience. 
+<details>
+
+<summary>Click here to display the full AT-TLS configuration file.</summary>
+
+```pagent
+# Main inbound rule, all API ML core services have it defined.
```

Comment:
This rule applies to more than just APIML.
References here should be "Zowe" such as "ZoweServerEnvironmentAction"
Because 7556 and 7557 are in this range but are not APIML.

Reply:
I'll rename these, thanks